ichnaea/data/tests/test_export.py

```diff
@@ -167,6 +181,7 @@ def test_upload(self, celery, session, metricsmock):
         reports.extend(self.add_reports(celery, 1, source="gnss"))
         reports.extend(self.add_reports(celery, 1, api_key="e5444e9f-7946"))
         reports.extend(self.add_reports(celery, 1, api_key=None, source="fused"))
+        reports.extend(self.add_reports(celery, 1, set_position=False))
```

Comment:
This test confirms the `geosubmit` exporter does not have an issue with `position: None`.